v23.1/export.md

```diff
@@ -5,6 +5,10 @@ toc: true
 docs_area: reference.sql
 ---
 
+{{site.data.alerts.callout_info}}
```

Comment:
I'd guess this is to head off questions but a part of me is like "why say this" if it's _not_ needed.  However I'm sure people will ask / are asking

Reply:
I agree, I think this maybe was added when it was no longer required and now remains.... I don't feel super confident about removing as part of this PR